clr.py

```diff
@@ -387,7 +430,9 @@ def on_epoch_end(self, epoch, logs=None):
             np.save(lrs_path, self.lrs)
 
             if self.verbose:
-                print("\tLR Finder : Saved the losses and learning rate values in path : {%s}" % (self.save_dir))
+                print(
```

Comment:
Put the first line on the same level as print. This new line corrects exactly one char spacing.